Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM docker-upgrade.artifactory.build.upgrade.com/python-base:20180919-4
+LABEL maintainer=devops@upgrade.com
+
+USER root
```

Comment:
Since we don't install python anymore.. do you still need root for the pip install?

Reply:
I just double check with upgrade user and it failed in downloading pip3 package. so it needs to be root just for pip installation 
``` PermissionError: [Errno 13] Permission denied: '/usr/local/lib/python3.7' ```
I guess because the python installed by root user